homeassistant/components/lutron_caseta/fan.py

```diff
@@ -61,27 +42,17 @@ class LutronCasetaFan(LutronCasetaDevice, FanEntity):
     """Representation of a Lutron Caseta fan. Including Fan Speed."""
 
     @property
-    def speed(self) -> str:
```

Comment:
Since Lutron seems to work with presets, should we implement `preset_mode` ?

Reply:
All the values are actually speeds (no auto or smart) so preset mode isn't needed in this case.